main/test/flix/Test.Handler.ZeroShot.AtomicInteger.flix

```diff
@@ -0,0 +1,163 @@
+mod Test.Handler.ZeroShot.AtomicInteger {
+
+    type alias AtomicInt = ##java.util.concurrent.atomic.AtomicInteger
```

Comment:
I have 3 OCD changes, I would request:

- Rename the type alias from AtomicInt to AtomicInteger.
- Add a helper function `newAtomicInteger` right after the type alias and use it below.
- Add a helper function `toInt` right after the above function and use it below.